scripts/typescript_checks.py

```diff
@@ -145,6 +145,7 @@
     'core/templates/filters/string-utility-filters/capitalize.filter.spec.ts',
     'core/templates/filters/string-utility-filters/capitalize.filter.ts',
     'core/templates/filters/string-utility-filters/convert-to-plain-text.filter.ts',
+    'core/templates/filters/string-utility-filters/convert-to-plain-text.filter.spec.ts',
```

Comment:
Please correctly type this file instead of adding it to the ignore list

Reply:
![Screenshot from 2022-10-15 09-39-00](https://user-images.githubusercontent.com/77006897/195968090-2647337d-2fc6-46af-85fd-00dbeffd605c.png)


if I am not adding this file to this list then I am getting lots of type errors from other files PTAL. this is the newly added unit test file in the code base `convert-to-plain-text.filter.spec.ts` earlier its testing was not covered properly.

Reply:
In general, you are responsible for typing any files you add and fixing any typing errors your PR introduces. If you want an exception to this rule, you'll need to ask vojtechjelinek since he's in charge of this typing project

Reply:
@vojtechjelinek PTAL

Reply:
As this file is newly added but as per offline discussion with @heyimShivam it is necessary step for migration to add this **AngularJs** file, so giving permission for this.

CC: @U8NWXD @vojtechjelinek.